### PR TITLE
Update integrations strategy for July 2022

### PR DESCRIPTION
**Pull request by ryankscott**
Updates the integrations strategy based on feedback and discussions.

### CODE REVIEW

File: content/strategy-goals/strategy/integrations/index.md [hunk 1]
```diff
@@ -6,103 +6,70 @@ Quick links:
...
+- Deprecating Sourcegraph extensions
+  - Prevent authoring of new extensions on our Sourcegraph.com extension registry
+  - Move “code intelligence” features to core functionality
+  - Move “git extras” (i.e. git blame etc.) extension to core functionality
+  - Move “open in editor” extension to core functionality
 
-- What are the key pain points for our customers that extensions could solve?
-- What are the key integrations that customers would like to see?
-- What do we have to add to our platform to support this?
+#### FY’23 Q4+
```
Comment (malomarrec) [inline content/strategy-goals/strategy/integrations/index.md:59]: I really like this framing of  "next 3 months" and then "later". might copy it

File: content/strategy-goals/strategy/integrations/index.md [hunk 2]
```diff
@@ -1,108 +1,73 @@
...
+#### Moving extensions into the core workflow
 
-We also believe that we can greatly improve the adoption of our IDE extensions by focusing on the management and rollout of extensions at scale. Often enterprises have supported IDEs, IDE extensions, or tooling for particular IDEs. If we can become a standard approved extension for IDEs at large companies, we can greatly increase our reach inside of them. To achieve this we’ll focus on making it easier to automatically configure and authenticate our extensions for private instances.
+The current goal for the company is to focus on the core workflow and developer time saved. The integrations team will contribute by moving the most heavily used extensions to be core parts of our product (e.g. code intelligence, open in editor, git extras etc.). For our customers this will result in these features being both faster and more discoverable.
```
Comment (jjinnii) [inline content/strategy-goals/strategy/integrations/index.md]:
```suggestion
To increase the speed and discoverability of our extension functionality, we will move the most heavily used extensions to be core parts of our product (e.g. code intelligence, open in editor, git extras etc.)
```

Comment (jjinnii) [reply content/strategy-goals/strategy/integrations/index.md]: Just a suggestion. We could add a second line like "This allows us to contribute to the company strategy of focusing on core workflows." or something along those lines if you want it to connect more to later parts of this page.

File: content/strategy-goals/strategy/integrations/index.md [hunk 3]
```diff
@@ -1,108 +1,73 @@
...
 
-#### Use Browser Extensions as a funnel for advanced functionality
+To further simplify the product and have a greater focus on the core workflow we will also deprecate the current Sourcegraph extensions and the extension registry. With the most used extensions becoming core parts of our product there's less need for our extensions. Instead, we will work towards a different extension model in order to support our goal of introducing new concepts to Sourcegraph. Deprecating Sourcegraph extensions will require us to work with our customers to determine what custom extensions have been built and endeavour to add them as native features.
 
-When launched, our Browser Extensions provided much needed code intelligence functionality into code hosts, however, this is now implemented natively (either directly in the case of GitHub, or indirectly through Web IDEs).
+#### Start introducing new concepts to Sourcegraph
```
Comment (jjinnii) [inline content/strategy-goals/strategy/integrations/index.md]:
```suggestion
#### Introduce new concepts to Sourcegraph
```

File: content/strategy-goals/strategy/integrations/index.md [hunk 4]
```diff
@@ -1,108 +1,73 @@
...
-
-Integrations align directly with our FY’23 strategy as some of our use cases require information from other tools (e.g. Managing security vulnerabilities). Integrations can also provide additional value to our existing features (e.g. Code Insights aggregating third party data with code search data).
+### Themes
 
-Integrations can also increase our addressable market by providing value for roles we don’t normally support. This could work in two ways, combining code with other third party data they typically use (e.g. Mapping code to infrastructure for SREs) or bringing visibility of Sourcegraph functionality to tools that other roles use (e.g. linking Sourcegraph notebooks in a Jira ticket for Product Managers). This aligns with early signals from our tech partners who are looking to expand their solutions beyond their current target market to developers.
+For the next 6 months, we will be focussing on the following two themes:
 
-#### Focus on bringing a great Sourcegraph search experience to more IDE users
+1. Moving extensions into the core workflow
```
Comment (jjinnii) [inline content/strategy-goals/strategy/integrations/index.md]:
```suggestion
1. Move extensions into the core workflow
```

File: content/strategy-goals/strategy/integrations/index.md [hunk 5]
```diff
@@ -1,108 +1,73 @@
...
+2. Introducing new concepts to Sourcegraph
```
Comment (jjinnii) [inline content/strategy-goals/strategy/integrations/index.md]:
```suggestion
2. Introduce new concepts to Sourcegraph
```

File: content/strategy-goals/strategy/integrations/index.md [hunk 6]
```diff
@@ -1,108 +1,73 @@
...
+In the long term we want Sourcegraph to become the primary source for information relating to code, tying all tools that developers use into a single platform. This will make Sourcegraph indispensable as we become the best way to answer any question relating to code, unblocking developers and saving their time.
 
-Because of the complexities of adding functionality directly into code hosts, we believe the best course of action is to continue to solve our core use cases (Code Reuse and Developer Onboarding) via Browser Extensions but redirect users to our web app for more advanced functionality. This approach has worked well with “Find References” and we believe we can continue to use it for other features.
+The first concept we want to introduce to Sourcegraph is code ownership. This has been a highly requested feature with clear, validated use cases across all of our products.
```
Comment (jjinnii) [inline content/strategy-goals/strategy/integrations/index.md:28]: Should we add a comment here about how we're going to spend time in discoveryt/technical discovery (or is that obvious)?

**Review by jjinnii — APPROVED** [review]
LGTM

File: content/strategy-goals/strategy/integrations/index.md (hunk 6 above)
Comment (ryankscott) [reply content/strategy-goals/strategy/integrations/index.md:28]: It's sort of implicit and it's also mentioned in the roadmap. I'm conscious of making this too wordy 😓

File: content/strategy-goals/strategy/integrations/index.md (hunk 2 above)
Comment (ryankscott) [reply content/strategy-goals/strategy/integrations/index.md]: Reworded this which hopefully does the same job.

**Pull request merged**